delivery_auto_refresh/tests/test_delivery_auto_refresh.py

```diff
@@ -273,6 +273,7 @@ def test_auto_refresh_so_and_unlink_line(self):
         self.assertFalse(delivery_line.exists())
 
     def test_auto_add_delivery_line_add_service(self):
+        self.env["ir.config_parameter"].sudo().set_param(self.param_name1, 1)
```

Comment:
Why do you need to do this?

Reply:
Hello, without this line the main purpose of the module (automatically refreshing the carrier in the SO) won't be enabled, so the test would always pass without going through the functionality of the module itself.

This is actually the main reason why this was an already addressed issue, but it broke again since the test wasn't accurate.

Let me know if you any more doubts so i can be more clear :)